### DISCUSSION TITLE
Add Horizonal Rules to Markup Support

### DISCUSSION THREAD

**Discussion by Katerlad** [GitbookIO/community#74] [Feature Requests]
### What's your feature/idea?

Please describe here - in as much detail as you can - your idea.

Please add Horizontal Rules to the Markup Language support. 

Horizontal Rules are shown in your advertisements but are not provided as a feature currently.

### What's your desired outcome?

What would you like to be able to do, if your idea came to life?

Use horizontal rules to add organization to headers with a cleaner/sleeker look like in your advertisements.

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [X] Medium
- [ ] Low

### Anything else?

![image](https://user-images.githubusercontent.com/54480322/216694485-7a06a99b-e160-467c-91d2-739f1eed68f3.png)

Here is an example of Horizontal Rules in Markup.

![image](https://user-images.githubusercontent.com/54480322/216694885-06c2ef67-87f8-47d3-9bd0-4ccb2d622c8f.png)

Here is the Horizontal rules you guys false advertise  :lau

**Comment by kristenoneill** [comment]
@Katerlad - I hacked this by creating a table (1 cell) with no content - not ideal - but working for me for now and thought I 'd share. This page is still a work in progress but you can see it [here](https://handbook.canvasmedical.com/productupdates/).
